frontend/cypress/pages/metrics/report.ts

```diff
@@ -22,6 +22,10 @@ class Report {
     return cy.contains('Previous')
   }
 
+  get saveButton() {
+    return cy.contains('Save')
```

Comment:
It is not recommended to use only text to match buttons. The Save text is too common, and it is easy to match errors.